Makefile

```
@@ -1,5 +1,19 @@
-test: vet
-	go test ./...
+DIR := ${CURDIR}
+build_dir := $(DIR)/.build
+golangci_lint_version = v1.24.0
+golangci_lint_dir = $(build_dir)/golangci_lint/$(golangci_lint_version)
+golangci_lint_bin = $(golangci_lint_dir)/golangci-lint
 
-vet:
-	go vet ./...
+$(golangci_lint_bin):
+	@echo "Installing golangci-lint $(golangci_lint_version)..."
+	$(E)rm -rf $(dir $(golangci_lint_dir))
```

The `(E)` stuff isn't wired up in this makefile. I'd remove it (on this line and others):
```suggestion
	@rm -rf $(dir $(golangci_lint_dir))
```